src/features/game/components/modal/components/VIPItems.tsx

```diff
@@ -310,11 +310,19 @@ export const VIPItems: React.FC<Props> = ({ onClose, onSkip }) => {
               text: t("vip.benefit.competition"),
               icon: trophyIcon,
             },
+            ...(getCurrentSeason() === "Winds of Change"
+              ? [
+                  {
+                    text: t("vip.benefit.windOfChange"),
+                    icon: increaseArrow,
```

Comment:
Consider using one of the new crop icons so they understand what new means.

The Brocolli would be a good one to use here